manifests/init.pp

```diff
@@ -12,6 +12,8 @@
 #
 # $enable_deb::                                Enable debian content plugin
 #
+# $enable_ansible::                            Enable ansible content plugin
```

Comment:
```suggestion
# $enable_ansible::                            Enable ansible content plugin to sync and serve ansible content
```

I had this feedback originally on the puppet-katello PR but it makes more sense here. The reason is that this is what users will see when looking at the output of `foreman-installer --help`

The implementation you have is consistent with how the other content plugin params are already documented. I think there is a potential clarity issue with ansible content specifically because we support many other features with ansible (REX, adding roles to hosts/hostgroups, we provide ansible collections to automate operations and configuration of Foreman/Katello).

If we think there is value in it (keeping the doc strings consistent for all of the content plugin params), I can tackle this in a separate PR which updates the other content plugin param docs as well. I don't care how it gets done, I just want to make sure it's crystal clear for users what this does.

Reply:
In fact I will go ahead and do that, so don't worry about it here :)